.github/workflows/vercel-deploy-erd.yml

```diff
@@ -75,10 +75,6 @@ jobs:
         run: pnpm exec vercel pull --yes --environment=${{ needs.setup-deployment.outputs.environment }} --token=${{ secrets.VERCEL_TOKEN }}
       - name: Pull Vercel Enviroment Variables
         run: pnpm vercel:env-pull --yes --environment=${{ needs.setup-deployment.outputs.environment }} --token=${{ secrets.VERCEL_TOKEN }}
-      - name: Run prepare command
-        run: pnpm build
-      - name: Update index.html content
-        run: pnpm --filter ${{ matrix.apps.name }} update_dist_content
```

Comment:
Perhaps the reason it was run twice is because of this update_dist_content.
This was done for @liam-hq/erd-sample, but is no longer needed now that we have moved to building on Vercel.